src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -220,6 +228,7 @@ public String generateToolTip(CategoryDataset dataset, int row,
         ar.setSeriesPaint(0, ColorPalette.RED);    // error
         ar.setSeriesPaint(1, ColorPalette.BLUE);   // info
         ar.setSeriesPaint(2, ColorPalette.YELLOW); // warning
+        ar.setSeriesPaint(3, ColorPalette.BLUE);   // debug
```

Comment:
Even if adding debug was a good idea (I don't think it is), it doesn't really make sense for it to default to the same colour as another built-in status (info).